docs/roundabout.md

```diff
@@ -1,14 +1,20 @@
 # Roundabout
 
-> Roundabout allows the creation of short lived presigned URLs for content stored in carpark bucket (in R2). It can also get the HTTP request redirected to the given presigned URL.
+> Roundabout allows the creation of short lived presigned URLs for content stored in web3.storage buckets (in R2). It can also get the HTTP request redirected to the given presigned URL.
 
 ## HTTP API
 
 The given API is currently public.
 
 ### `GET /{carCid}`
 
-Redirects to a URL where the requested CAR file (by its CID) can be downloaded from. The request will return a `302 Redirect` to a created presigned URL.
+Redirects to a presigned URL where the requested CAR file (by its CID) can be downloaded from. This will use web3.storage `carpark` as the location of the requested CARs. The request will return a `302 Redirect` to a created presigned URL.
+
+It also supports a query parameter `expires` with the number of seconds this presigned URL should be valid for. You can set a value from one second to 7 days (604,800 seconds). By default the expiration is set for 3 days (259,200 seconds).
+
+### `GET /raw/{key}?bucket=bucket-name`
```

Comment:
Given that you can use `GET /raw/{key}` to do the same thing as `GET /{carCid}` like `GET /raw/bagbaiera222226db4v4oli5fldqghzgbv5rqv3n4ykyfxk7shfr42bfnqwua/bagbaiera222226db4v4oli5fldqghzgbv5rqv3n4ykyfxk7shfr42bfnqwua.car` I'd be tempted to simplify and just have one endpoint `GET /{key}`.

Reply:
Yeah, I wrote this in original post. My goal to not do this is to not delegate responsability to know key format to the client, because in future we can have multiple buckets (with sharding or other indexing ways). This `/raw/{key}` seems more like a temporary exception for the real goal of roundabout.

I am happy to change to single route if you think that makes more sense that my reasoning written in original post

Reply:
ACK, makes sense! Lets ship what's here but please lets restrict the bucket.